src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
+import duke.*;
```

Comment:
Would it be better to list all the imported classes?

Reply:
I concur with @gordonlzy. I think this violates one of the [coding standards.](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)

Reply:
I agree as well. Imports should be one at a time.